version/version.go

```diff
@@ -4,4 +4,5 @@
 
 package version
 
-const defaultBeatVersion = "8.9.0"
+// FIXME: for testing only
```

Comment:
If this is only for testing, it should be removed, right ?

Reply:
Removed.